db/seeds.rb

```diff
@@ -12,3 +12,19 @@
 end
 
 user.update_attribute(:confirmed_at, Time.now) if user.confirmed_at.nil?
+
+company = Company.create!(name: 'Test Company')
+
+company.company_users.create(user: user)
+
+teams = Team.create!([
+  { name: "Diamond", company: company },
+  { name: "Ruby", company: company },
+  { name: "Cristal", company: company },
+  { name: "Green", company: company },
+  { name: "Opal", company: company }
+])
+
+team_user = TeamUser.create!([
```

Comment:
seeds should have at least two teams each with team lead and just team members

Reply:
fixed